# Hypothesis Testing Q4 

 TeleCall uses 4 centers around the globe to process customer order forms. They audit a certain %  of the customer order forms. Any error in order form renders it defective and has to be reworked before processing.  The manager wants to check whether the defective %  varies by centre. Please analyze the data at 5% significance level and help the manager draw appropriate inferences


# 1 - Business Problem
Does the defective % varies significantly by centre ? 

# 2 - Data description
α == 0.05 (95% Confidence)

Y == Discrete
X == Discrete

Since there are more than 2 variable we will perform Chi-Square test 

# 3 - Import Libs/Data 

In [2]:
import pandas as pd
import numpy as np
from scipy import stats
from scipy.stats import norm
from scipy.stats import chi2_contingency

In [3]:
telecall_data = pd.read_csv('Costomer+OrderForm.csv')
telecall_data

Phillippines   Indonesia       Malta       India
0     Error Free  Error Free   Defective  Error Free
1     Error Free  Error Free  Error Free   Defective
2     Error Free   Defective   Defective  Error Free
3     Error Free  Error Free  Error Free  Error Free
4     Error Free  Error Free   Defective  Error Free
..           ...         ...         ...         ...
295   Error Free  Error Free  Error Free  Error Free
296   Error Free  Error Free  Error Free  Error Free
297   Error Free  Error Free   Defective  Error Free
298   Error Free  Error Free  Error Free  Error Free
299   Error Free   Defective   Defective  Error Free

[300 rows x 4 columns]

#### H0 == The defective % does not varies by centre
#### H1 == The defective % does varies by centre

# 4 - Chi-Square Test

In [6]:
stats,chi2_contingency([telecall_data['Phillippines'].value_counts(), telecall_data['Indonesia'].value_counts(), telecall_data['Malta'].value_counts(), telecall_data['India'].value_counts()])

(<module 'scipy.stats' from 'C:\\Users\\shubham\\anaconda3\\lib\\site-packages\\scipy\\stats\\__init__.py'>,
 (3.8589606858203545,
  0.2771020991233144,
  3,
  array([[271.75,  28.25],
         [271.75,  28.25],
         [271.75,  28.25],
         [271.75,  28.25]])))

##### P value of Chi-Square test == 0.277 > α

### Thus HO is accepted.
### The defective % does not varies significantly by centres
 